packages/tizen_rpc_port/lib/src/rpc_port_method_channel.dart

```diff
@@ -0,0 +1,42 @@
+// Copyright 2022 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file
+
+import 'package:flutter/services.dart';
+
+// ignore_for_file: public_member_api_docs
+class MethodChannelRpcPort {
+  static final MethodChannelRpcPort instance = MethodChannelRpcPort();
+
+  final MethodChannel _channel = const MethodChannel('tizen/rpc_port');
+
+  Stream<dynamic>? _stream;
```

Comment:
Please add a new stream member variable to this class and separate the stream from proxyConnect() and the stubListen().

The application can be both a proxy and a stub of the rpc-port.

Reply:
Thank you.
I applied your suggestion.